database/sql/V057__add_fee_policy.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TYPE PolicyKind AS ENUM ('priceimprovement', 'volume');
```

Comment:
We have a [README](https://github.com/cowprotocol/services/blob/main/database/README.md) for the database.
Would be good to make a habit out of updating that whenever we add a new DB migration.

Reply:
https://github.com/cowprotocol/services/pull/2118/commits/22f279fc34c7ffbbd7421ca393427e81b5a892a3